website/docs/diff.md

```diff
@@ -14,6 +14,7 @@ sidebar_position: 2
    * database name cannot contain capital letters.
 4. For Tigris, FerretDB requires Tigris schema validation for `msg_create`: validator must be set as `$tigrisSchemaString`.
    The value must be a JSON string representing JSON schema in [Tigris format](https://docs.tigrisdata.com/overview/schema).
+5. FerretDB requires document keys to be valid UTF-8 strings and not contain `$` sign.
```

Comment:
MongoDB requires UTF-8, so that's not a difference.

I also think that should be a point number 3, to group similar things together

Reply:
Point number three seems to be about dbs and collections, and it's already quite long (and gives some cognitive load to a reader), added point number four. But feel free to rephrase if you want.

Reply:
fixed

Reply:
Does the no. 5 difference only apply to Tigris?